pages/trips/[id].js

```diff
@@ -15,15 +17,18 @@ export default function DetailsPage() {
     router.push("/");
   }
   return (
-    <TripDetails
-      image={trip.image}
-      title={trip.title}
-      city={trip.city}
-      country={trip.country}
-      startDate={trip.startDate}
-      endDate={trip.endDate}
-      description={trip.description}
-      handleDeleteTrip={handleDeleteTrip}
-    />
+    <>
+      <Link href={`/editTrip/${id}`}>Edit Trip</Link>
```

Comment:
I would call this route `/trips/${id}/edit`

You'll have to create it like:

```
trips
  [id].js
  edit
     [id.js]
```

Reply:
as recommended in our recap we changed the structure like:

trips
      [id]
       edit.js
       index.js